papaya-web/src/main/java/pl/edu/agh/papaya/rest/projects/service/SprintsRestService.java

```diff
@@ -13,29 +12,40 @@
 import org.springframework.stereotype.Service;
 import pl.edu.agh.papaya.api.model.SprintDto;
 import pl.edu.agh.papaya.api.model.SprintStateDto;
+import pl.edu.agh.papaya.api.model.SprintSummaryDto;
+import pl.edu.agh.papaya.api.model.UserAvailabilityDto;
 import pl.edu.agh.papaya.mappers.LocalDateTimePeriodMapper;
 import pl.edu.agh.papaya.mappers.SprintMapper;
 import pl.edu.agh.papaya.mappers.SprintStateMapper;
 import pl.edu.agh.papaya.model.LocalDateTimePeriod;
 import pl.edu.agh.papaya.model.Project;
 import pl.edu.agh.papaya.model.Sprint;
 import pl.edu.agh.papaya.model.SprintState;
+import pl.edu.agh.papaya.security.UserContext;
 import pl.edu.agh.papaya.service.sprint.SprintService;
 import pl.edu.agh.papaya.util.AssertionUtil;
+import pl.edu.agh.papaya.util.ForbiddenAccessException;
 import pl.edu.agh.papaya.util.ResourceNotFoundException;
 
 @Service
 @RequiredArgsConstructor
+@SuppressWarnings({"ClassFanOutComplexity"})
```

Comment:
Chyba przechodzi bez tego suppressa wszystko

Reply:
Ciężko uwierzyć, że się aż tak udało to odchudzić . Usuwam